errors/placeholder-blur-data-url.md

```diff
@@ -0,0 +1,15 @@
+# `placeholder=blur` without `blurDataURL`
+
+#### Why This Error Occurred
+
+You are attempting use the `next/image` component with `placeholder=blur` property but no `blurDataURL` property.
+
+The `blurDataURL` might be missing because your using a string for `src` instead of a static import.
+
+Or `blurDataURL` might be missing because the static import is an unsupported image format.
```

Comment:
```suggestion
Or `blurDataURL` might be missing because the static import is not one of the automatic blur supported types.
```

Is there another PR we are adding the supported image types for automatic blur in the docs we could point to here?